### PR TITLE
Adding standard elliptic curve usage behaviour to console test

### PR DESCRIPTION
**Pull request by nickrmc83**
Changed the console tests to use separate digest and sign functions for ECDSA named curves. The previous implementation used some non-standard mechanisms to achieve the same.

The build of this pull requires updates to typings/pkcs11js

### CODE REVIEW

**Comment by rmhrisk** [comment]
It seems the build is failing:

image

**Comment by nickrmc83** [comment]
The failing build is due to the need to update the pkcs11js typings.

**Comment by rmhrisk** [comment]
@microshine please update the pkcs11js typings

File: vendor/ThalesNShield.json [hunk 1]
```diff
@@ -27,5 +27,37 @@
   "WRAP_RSA_CRT_COMPONENTS" : 3728959859,
   "CAC_TK_DERIVATION" : 3728959860,
   "NC_AES_CMAC_KEY_DERIVATION" : 3728960626,
+<<<<<<< HEAD
```
Comment (nickrmc83) [inline vendor/ThalesNShield.json]: This has been resolved in the latest merge.

**Comment by nickrmc83** [comment]
@rmhrisk is there anything you need from me to push this change through?

**Pull request merged**